bindings/python/src/expose-settings.hpp

```diff
@@ -71,6 +71,8 @@ exposeSettings(pybind11::module_ m)
                    &Settings<T>::compute_preconditioner)
     .def_readwrite("update_preconditioner", &Settings<T>::update_preconditioner)
     .def_readwrite("check_duality_gap", &Settings<T>::check_duality_gap)
+    .def_readwrite("eps_gap_abs", &Settings<T>::eps_gap_abs)
+    .def_readwrite("eps_gap_rel", &Settings<T>::eps_gap_rel)
```

Comment:
Would be better to rename it fully as follows:
```suggestion
    .def_readwrite("eps_duality_gap_abs", &Settings<T>::eps_gap_abs)
    .def_readwrite("eps_duality_gap_rel", &Settings<T>::eps_gap_rel)
```

Reply:
Done in b480bc26c1d14b0d4c4a226968327f2e2c9a95d2